src/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -649,8 +650,10 @@ private bool AcquireServerCredentials(ref byte[] thumbPrint, byte[] clientHello)
             }
             else if (_sslAuthenticationOptions.CertSelectionDelegate != null)
             {
-                X509CertificateCollection tempCollection = new X509CertificateCollection();
-                tempCollection.Add(_sslAuthenticationOptions.ServerCertificate);
+                X509CertificateCollection tempCollection = new X509CertificateCollection
+                {
+                    _sslAuthenticationOptions.ServerCertificate
+                };
```

Comment:
Not sure this is an improvement.

Reply:
Was just following the `.editorconfig` https://github.com/dotnet/corefx/blob/963e0d64bd7a69ba4b8d4d933bd56d0f2a5a8913/.editorconfig#L89 will revert

Reply:
Thanks.  We need to revise that config file.  But it's also appreciated to avoid applying any refactorings for style in PRs intended for other purposes; makes it hard to find the signal in the noise.